opendebates/templates/base.html

```diff
@@ -70,8 +70,8 @@
       {% endif %}
     </div>
     <div class="header-title">
-      <h1><a href="/">{{ SITE_THEME.HEADER_TITLE|linebreaks }}</a></h1>
-      <div class="header-copy">{{ SITE_THEME.HEADER_COPY }}</div>
+      <h1><a href="/">{{ BANNER_HEADER_TITLE|safe }}</a></h1>
+      <div class="header-copy">{{ BANNER_HEADER_COPY }}</div>
```

Comment:
Might as well run this one through `safe` as well so the two fields behave the same. The primary purpose I can see for having markup in the title is to allow line breaks for styling (or colors as you mention), and I can see that being an issue for the copy as well.